contrib/nsqadmin.cfg.example

```diff
@@ -1,6 +1,42 @@
 ## log verbosity level: debug, info, warn, error, or fatal
 log_level = "info"
 
+## log message prefix (default "[nsqadmin] ")
+# log_prefix = ""
+
+## HTTP header to check for authenticated admin users (default "X_Forwarded_User")
+# acl_http_header = ""
+
+## admin user (may be given multiple times; if specified, only these users will be able to perform privileged actions; acl_http_header is used to determine the authenticated user)
+# admin_users = [
+#     "admin",
+#     "asdqwc124134134e"
```

Comment:
this is an odd example username

Reply:
i consider "admin_users" as a secret string actually, I'll change it.

Reply:
Ah ... that's a clever way to use it! (As you probably already know, it is intended to be used behind an auth proxy, which is trusted to always set or clear this header value, like https://github.com/oauth2-proxy/oauth2-proxy)